libs/features/components/cards/src/lib/pages/elewa-group-card/elewa-group-card.component.html

```diff
@@ -0,0 +1,26 @@
+<div class="row row-cols-1 row-cols-md-3 g-4">
```

Comment:
This should not be in a pages folder as it's not a page, also the card is already present. Please sync with the latest changes and work from there.